chemreg/substance/models.py

```diff
@@ -43,3 +43,20 @@ class SynonymType(ControlledVocabulary):
 
     validation_regular_expression = models.TextField(blank=True)
     score_modifier = models.FloatField(default=0)
+
+
+class SynonymQuality(ControlledVocabulary):
+    """Controlled vocabulary for SynonymQuality
+
+    Attributes:
+        Name = String (Less than 50 character, url safe, unique, required field)
+        Label = String (Less than 100 characters, unique, required field)
+        Short Description = String (Less than 500 characters, required field)
+        Long Description = TEXT (required field)
+        score_weight = Float (default 1.0) greater than 0
+        is_restrictive = Boolean
+    TODO: Add Validation in Serializer for score_weight, unable to set minimum value for FloatField's
+    """
+
+    score_weight = models.FloatField(default=1.0)
+    is_restrictive = models.BooleanField
```

Comment:
```suggestion
    is_restrictive = models.BooleanField()
```
this class def needs to be called to instantiate the field
```
django.core.exceptions.FieldDoesNotExist: SynonymQuality has no field named 'is_restrictive'
```